README.markdown

```diff
@@ -217,6 +217,8 @@ by putting the simple type name into `type` field, and the value in the `value`
 {"value":{"name":"Company","value":2},"type":"Organization"}
 ```
 
+Note that both `unionFormatN` and `flatUnionFormatN` functions only support leaf types in `A1`, `A2`, etc, so if your ADT consists of multiple levels of traits, you need to expand them yourself to a flat list of concrete types.
```

Comment:
"Note" is too tame for "this will break at runtime". "BEWARE" or similar is better.

Also, could you document this in the scaladocs of `unionFormatN` and `flatUnionFormatN` too, please? To give it the best chance to be noticed, understood and worked around.